src/style.css

```
@@ -0,0 +1,46 @@
+* {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+}
+
+.stars {
+  display: flex;
+  position: relative;
+  align-items: flex-start;
+  padding: 2px;
+}
+
+.stars__star {
+  position: relative;
+  height: 12px;
+  width: 13px;
+  margin-right: 7px;
+
+  background-image: url(images/star.svg);
+  background-size: cover;
+}
+
+.stars--1 > .stars__star:nth-child(1) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--2 > .stars__star:nth-child(-n + 2) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--3 > .stars__star:nth-child(-n + 3) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--4 > .stars__star:nth-child(-n + 4) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--5 > .stars__star:nth-child(n) {
+  background-image: url(images/star-active.svg);
+}
```

That's what I meant:

```suggestion
.stars--1 > .stars__star:nth-child(1),
.stars--2 > .stars__star:nth-child(-n + 2),
.stars--3 > .stars__star:nth-child(-n + 3),
.stars--4 > .stars__star:nth-child(-n + 4),
.stars--5 > .stars__star:nth-child(n) {
  background-image: url(images/star-active.svg);
}
```